<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab6</class>
 <widget class="QMainWindow" name="Lab6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>1641</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Lab6</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="shapesTableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Shape type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Pen color</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Brush color</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Pen thickness</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1520</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Shape</string>
    </property>
    <addaction name="actionRectangle"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionTriangle"/>
    <addaction name="actionLine"/>
    <addaction name="actionImage"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionChoose_Color"/>
    <addaction name="actionSettings_dialog"/>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRectangle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionTriangle"/>
   <addaction name="actionLine"/>
   <addaction name="actionImage"/>
   <addaction name="separator"/>
   <addaction name="actionChoose_Color"/>
   <addaction name="actionReset"/>
   <addaction name="actionSettings_dialog"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/Images_lab6/Rect.ico</normaloff>:/Images_lab6/Rect.ico</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/Images_lab6/Circle.ico</normaloff>:/Images_lab6/Circle.ico</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionChoose_Color">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/Images_lab6/Colors.ico</normaloff>:/Images_lab6/Colors.ico</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionSettings_dialog">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/Images_lab6/Settings.ico</normaloff>:/Images_lab6/Settings.ico</iconset>
   </property>
   <property name="text">
    <string>  Settings Dialog</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/Images_lab6/Triangle.ico</normaloff>:/Images_lab6/Triangle.ico</iconset>
   </property>
   <property name="text">
    <string>Triangle</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/Images_lab6/Line.ico</normaloff>:/Images_lab6/Line.ico</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/Images_lab6/reset.ico</normaloff>:/Images_lab6/reset.ico</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="myresources.qrc">
     <normaloff>:/Images_lab6/image-outline.png</normaloff>:/Images_lab6/image-outline.png</iconset>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections/>
</ui>
